### PR TITLE
mail-client/mutt-wizard: version bump to 3.3.1

### PR DESCRIPTION
**Pull request by ephemer4l**
Dropped the ~x86 because I can't test on it currently. Added myself as maintainer.

Signed-off-by: Efe İzbudak efe.izbudak@metu.edu.tr

### CODE REVIEW

File: mail-client/mutt-wizard/mutt-wizard-3.3.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright 2019-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (juippis) [inline mail-client/mutt-wizard/mutt-wizard-3.3.1.ebuild:4]: Always aim to use the latest available EAPI, 8 should work with this ebuild.

File: mail-client/mutt-wizard/mutt-wizard-3.3.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+DESCRIPTION="A system for automatically configuring neomutt and isync"
+HOMEPAGE="https://github.com/LukeSmithxyz/mutt-wizard"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/LukeSmithxyz/mutt-wizard.git"
+else
+	SRC_URI="https://github.com/LukeSmithxyz/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
```
Comment (juippis) [inline mail-client/mutt-wizard/mutt-wizard-3.3.1.ebuild:16]: There's no need to drop existing arch support for simple version bumps. If there are major changes, ie API reworks or build system change, then you should drop the keywords to only what you can test. But normally just resetting the keywords to `~` will suffice on simple version bumps. Use **ekeyword ~all new-version.ebuild** to be sure!
https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades

Comment (ephemer4l) [reply mail-client/mutt-wizard/mutt-wizard-3.3.1.ebuild]: Fixed thank you!

File: mail-client/mutt-wizard/mutt-wizard-3.3.1.ebuild (hunk 1 above)
Comment (ephemer4l) [reply mail-client/mutt-wizard/mutt-wizard-3.3.1.ebuild]: Fixed. Thanks!

**Review by juippis — APPROVED** [review]
Thank you! Remember to sync updates to the live -9999 ebuild whenever you can.

**Pull request closed without merging**